SR2021/2021-04-27-finals-reminder.md

```diff
@@ -0,0 +1,50 @@
+---
+to: All SR2021 teams
+subject: Student Robotics Finals
+---
+
+Hi,
+
+Now that the league stage is over, we wanted to give you some quick reminders
+about the schedule for next weekend's knockouts. We'd also love to hear about
+about your experiences competing in SR2021.
+
+## Feedback & Demographics Survey
+
+To help us understand how we're doing we'd really like to understand the make up
+of your team and their experiences with the competition this year. If they
+haven't done so already please encourage your competitors to complete our
+[feedback survey][feedback-survey] before the knockouts.
+
+## Code Submission Deadline
+
+The deadline for [code submission][code-submitter] for the knockouts is 8pm BST
+(UK time) on Friday the 30th April. If your team is going to have any difficulty
+uploading your code by this time let us know before the deadline and we'll try
+to sort something out.
+
+*Note*: if you don't submit code for the knockouts we'll use the code you
```

Comment:
Should this explicitly say that teams who didn't enter league 4 aren't eligible? Or should we drop this section out for the ineligible teams? The latter feels nicer, if it's easy enough to do.

Reply:
Teams who didn't enter league 4 aren't in the knockouts anyway, so feels unnecessary to be explicit.

Reply:
Yes, we know that. However it doesn't stop them submitting code and thinking that they'll get entered.

Reply:
They know it too - it was in the [previous email](https://github.com/srobo/team-emails/blob/master/SR2021/2021-04-20-league-4-knockouts-simulator-update.md)

Reply:
True. Repetition is good however.